Parsers/ASimAuthentication/Parsers/ASimAuthenticationPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,167 @@
+                        DeviceCustomString1Label,
+                        DeviceCustomString1,
+                        DeviceCustomString3Label,
+                        DeviceCustomString3,
+                        DeviceCustomString4Label,
+                        DeviceCustomString4,
+                        DeviceCustomString5Label,
+                        DeviceCustomString5,
+                        DeviceCustomString6Label,
+                        DeviceCustomString6,
+                        "PanOSAuthenticationDescription",
+                        PanOSAuthenticationDescription,
+                        "PanOSClientTypeName",
+                        PanOSClientTypeName,
+                        "PanOSConfigVersion",
+                        PanOSConfigVersion,
+                        "PanOSMFAVendor",
+                        PanOSMFAVendor,
+                        "PanOSSourceDeviceCategory",
+                        PanOSSourceDeviceCategory,
+                        "PanOSSourceDeviceModel",
+                        PanOSSourceDeviceModel,
+                        "PanOSSourceDeviceProfile",
+                        PanOSSourceDeviceProfile,
+                        "PanOSSourceDeviceVendor",
+                        PanOSSourceDeviceVendor
+                    )
+      | project-rename
+          DvcIpAddr = Computer,
+          EventUid = _ItemId,
+          DvcId = DeviceExternalID,
+          EventOriginalResultDetails = Message,
+          EventOriginalSeverity = LogSeverity,
+          EventOriginalType = DeviceEventClassID,
+          EventOriginalUid = ExtID,
+          EventProductVersion = DeviceVersion,
+          LogonProtocol = PanOSAuthenticationProtocol,
+          SrcDvcOs = PanOSSourceDeviceOSFamily,
+          TargetUsername = PanOSAuthenticatedUserName,
+          TargetUserId = PanOSAuthenticatedUserUUID,
+          TargetDomain = PanOSAuthenticatedUserDomain,
+          EventOriginalSubType = Activity,
+          HttpUserAgent =  PanOSUserAgentString,
+          TargetDvcScopeId = PanOSCortexDataLakeTenantID,
+          TargetSessionId = PanOSSessionID
+      | extend
+          Dvc = coalesce(DvcFQDN, DvcId, DvcHostname, DvcIpAddr),
+          EventEndTime = EventStartTime,
+          EventResult = iff(EventMessage has "Invalid Certificate", "Failure", "Success"),
+          Dst = TargetIpAddr,
+          Src = coalesce(SrcFQDN, SrcHostname, SrcIpAddr),
+          TargetUserType = _ASIM_GetUserType(TargetUsername, ""),
+          User = TargetUsername,
+          IpAddr = SrcIpAddr,
+          DvcIdType = iff(isnotempty(DvcId), "Other", ""),
+          TargetDomainType = case(
+                        array_length(split(DestinationUserName, ".")) > 1, "FQDN",
+                        array_length(split(DestinationUserName, "\\")) > 1, "Windows",
+                        ""
+                    ),
+          TargetUserIdType = iff(isnotempty(TargetUserId), "UID", ""),
```

Comment:
@niralishah-crest 
UID is => A Linux user ID.
https://learn.microsoft.com/en-us/azure/sentinel/normalization-about-schemas#useridtype

Reply:
isn't this ID would be specific to Palo Alto?

Reply:
not changing. no sample available